app/frontend/src/cljs/teet/contract/contract_partners_view.cljs

```diff
@@ -498,17 +496,19 @@
      [:h1 partner-name]
      (when lead-partner?
        [common/primary-tag (tr [:contract :lead-partner])])
-     [buttons/button-secondary
-      {:href (routes/url-for {:page :contract-partners
-                              :params params
-                              :query {:page :edit-partner
-                                      :partner teet-id}})}
-      (tr [:buttons :edit])]]))
+     [authorization-check/when-authorized
+      :thk.contract/edit-contract-partner-company partner
+      [buttons/button-secondary
+       {:href (routes/url-for {:page :contract-partners
+                               :params params
+                               :query {:page :edit-partner
+                                       :partner teet-id}})}
+       (tr [:buttons :edit])]]]))
 
 (defn partner-info
   [e! {:keys [params] :as app} selected-partner]
   [:div
-   [partner-info-header selected-partner params]
+   [partner-info-header selected-p]
```

Comment:
"selected-p" is not very descriptive name. A suggest "selected-partner" instead

Reply:
nice catch I definitely didn't mean to change those params at all :D